src/containers/StudyEdit/style.ts

```diff
@@ -0,0 +1,44 @@
+import styled from '@emotion/styled';
+
+export const StudyEditWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 3rem 1rem;
+  gap: 2rem;
+`;
+
+export const StudyEditHeading = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+`;
+
+export const StudyEditImageWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+`;
+
+export const ImageWrapper = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: center;
+`;
```

Comment:
이렇게 수정하면 이미지를 보여주는 공간을 좀 더 명확히 보여줄 수 있을 것 같습니다!
```suggestion
export const ImageWrapper = styled.div`
  position: relative;
  width: 296px;
  height: 200px;
  border-radius: 0.5rem;
  border: 1px solid #d1d5db;
  align-items: center;
`;
```

Reply:
감사합니다 맥키😁 반영 완료했습니다! 27790bf6f41c5f10a07a56fe0013529d292e9caf